N3, Ebselen, and PPI networks (#193)
greenelab/covid19-review: content/20.treatments.md

#### Protease Inhibitors

The inhibition of virus proteases could be a promising approach to stop to the replication of the pathgen.

##### N3
N3 is an inhibitor of Mpro, a 33.8-kDa SARS-CoV-2 protease that is involved in viral replication and transcriptipn.

###### Anticipated Mechanism
N3 inhibits Mpro through binding to its substrate pocket.

###### Current Evidence

N3 was first designed computationally to bind in the substrate binding pocket of SARS-CoV and MERS-CoV Mpro proteases. 
Consequently, the structure N3-bound SARS-CoV-2 Mpro has been solved [@doi:10.1038/s41586-020-2223-y], which confirmed the computational prediction.
Finally, N3 reduced the viral in samples taken from patients.

###### Summary

N3 is a computationally designed molecule that inhibits the viral transcription through inhibiting Mpro.
Although N3 is a strong inhibitor of SARS-CoV-2 in vitro, its safety and efficacy have to be tested in healthy volunteers and patients.

##### Ebselen

> Ebselen identified as Mpro protease inhibitor. It is currently investigated as an anti-oxidant drug [@doi:10.1007/s11033-014-3417-x].

Reviewer: ```suggestion
Ebselen identified as Mpro protease inhibitor.
It is currently investigated as an anti-oxidant drug [@doi:10.1007/s11033-014-3417-x].
```

###### Anticipated Mechanism
Ebselen inhibits Mpro through binding to its substrate pocket.

###### Current Evidence

After the design and confirmation of N3 as a highly potent Michael acceptor inhibitor and the identification of Mpro structure [@doi:10.1038/s41586-020-2223-y], 10000 compounds were screened for their in vitro anti-Mpro activity.

> The six leads that were identified were Ebselen, Disulfiram, Tideglusib, Carmofour, PX-12. However when the compounds were further assayed on patient viral samples, Ebselen has the strongest potency in reducing the viral load.

Reviewer: ```suggestion
The six leads that were identified were Ebselen, Disulfiram, Tideglusib, Carmofour, PX-12.
However when the compounds were further assayed on patient viral samples, Ebselen has the strongest potency in reducing the viral load.
```

I've seen some skepticism over the hits from this high-throughput screen because they are mostly frequent hitters.  [Derek Lowe's blog post](https://blogs.sciencemag.org/pipeline/archives/2020/04/10/more-on-screening-for-coronavirus-therapies) summarizes some of this.

In the comments there, Brian Shoichet pulls out this line from the paper:
> In general, such molecules are expected to be promiscuous binders and therefore, as they stand, may have limited potential as drug leads.

Let's add something about the caveats of these six leads.

Author: makes sense, thanks @agitter I will add a few lines to reflect that.

#### Protease Inhibitors

The inhibition of virus proteases could be a promising approach to stop to the replication of the pathgen.

##### N3
N3 is an inhibitor of Mpro, a 33.8-kDa SARS-CoV-2 protease that is involved in viral replication and transcriptipn.

###### Anticipated Mechanism
N3 inhibits Mpro through binding to its substrate pocket.

###### Current Evidence

> N3 was first designed computationally to bind in the substrate binding pocket of SARS-CoV and MERS-CoV Mpro proteases. 

Reviewer: Is there a good reference for this statement?

Author: I just added two references in the last commit.

Consequently, the structure N3-bound SARS-CoV-2 Mpro has been solved [@doi:10.1038/s41586-020-2223-y], which confirmed the computational prediction.

> Finally, N3 reduced the viral in samples taken from patients.

Reviewer: ```suggestion
Finally, N3 reduced the viral load in samples taken from patients.
```

###### Summary

N3 is a computationally designed molecule that inhibits the viral transcription through inhibiting Mpro.
Although N3 is a strong inhibitor of SARS-CoV-2 in vitro, its safety and efficacy have to be tested in healthy volunteers and patients.

##### Ebselen
Ebselen identified as Mpro protease inhibitor.
It is currently investigated as an anti-oxidant drug [@doi:10.1007/s11033-014-3417-x].

###### Anticipated Mechanism
Ebselen inhibits Mpro through binding to its substrate pocket.

###### Current Evidence

> After the design and confirmation of N3 as a highly potent Michael acceptor inhibitor and the identification of Mpro structure [@doi:10.1038/s41586-020-2223-y.@doi:10.1126/science.abb3405], 10000 compounds were screened for their in vitro anti-Mpro activity.

Reviewer: ```suggestion
After the design and confirmation of N3 as a highly potent Michael acceptor inhibitor and the identification of Mpro structure [@doi:10.1038/s41586-020-2223-y; @doi:10.1126/science.abb3405], 10000 compounds were screened for their in vitro anti-Mpro activity.
```

Updating to semi-colon separate the multiple citations

The six leads that were identified were Ebselen, Disulfiram, Tideglusib, Carmofour, PX-12.

> However when the compounds were further assayed on patient viral samples, Ebselen has the strongest potency in reducing the viral load.

Reviewer: I like the new text below summarizing Ebselen.  In this sentence, perhaps it could be rephrased to something like
> However, the authors noted concerns about the potential promiscuity of these compounds.
> When the compounds were further assayed on patient viral samples, Ebselen has the strongest potency in reducing the viral load.

I'm not making this as a GitHub suggestion because we should go back to the original paper to see what specifically the authors said about these compound as promiscuous binders.

Author: Thanks for looking into this @agitter 
The authors state that ' In general, such molecules are expected to be promiscuous
binders and therefore, as they stand, may have limited potential as
drug leads.' 
Would like to make your addition as a github change?

Author: I just saw your commit, the wording is much better.
`However, the authors cautioned that these compounds are likely promiscuous binders, which would diminish their therapeutic potential.`

> However when the compounds were further assayed on patient viral samples, Ebselen has the strongest potency in reducing the viral load.

Reviewer: ```suggestion
When the compounds were further assayed on patient viral samples, Ebselen had the strongest potency in reducing the viral load.
However, the authors cautioned that these compounds are likely promiscuous binders, which would diminish their therapeutic potential.
```

@marouenbg do you like this phrasing?  Feel free to edit it further.

Author: I think this is much better, thank you.

#### Protease Inhibitors

> The inhibition of virus proteases could be a promising approach to stop to the replication of the pathgen.

Reviewer 2: Hmph. IMO, we should refrain from strong recommendations in a specific section. It may be better to keep each section more evidence-based and evaluate them collectively in the discussion section?

Author: Thanks @ypar for reviewing this and catching all the errors.
What would you suggest to have in the `why it may be useful?` section of` Protease Inhibitors`

Reviewer 2: pathgen -> pathogen

Reviewer 2: something along the lines of:

molecular studies showed that viral proteases play an important role in the life cycle of (corona)viruses by modulating the cleavage of viral polyprotein precursors, etc. FDR approved drugs already exist (e.g. HIV, HepC, ...). some evidence shows that a cocktail of protease inhibitors and immune boosters could be beneficial for sars-cov-2. some evidence showed a type of protease inhibitor worked for sars and mers... etc.
but limitations include non-randomized trials used for some of these evaluations, many existing inhibitors are for retrovirals and cov is not.. etc.

Author: I tried to develop those ideas in 24b4357. @ypar please let me know if they make sense.

##### N3

> N3 is an inhibitor of Mpro, a 33.8-kDa SARS-CoV-2 protease that is involved in viral replication and transcriptipn.

Reviewer 2: transcriptipn -> transcription

Author: fixed in #660b45f

###### Anticipated Mechanism
N3 inhibits Mpro through binding to its substrate pocket.

###### Current Evidence

> N3 was first designed computationally [@doi:10.1371/journal.pbio.0030428] to bind in the substrate binding pocket of SARS-CoV and MERS-CoV Mpro proteases [@doi:/10.1007/s13238-013-2841-3]. 

Reviewer: ```suggestion
N3 was first designed computationally [@doi:10.1371/journal.pbio.0030324] to bind in the substrate binding pocket of the Mpro protease of SARS-like coronaviruses [@doi:10.1007/s13238-013-2841-3]. 
```

Great, thanks for finding these.  I updated the first one.  I believe that `pbio.0030428` points to the correction, not the original article.

I reworded the second part because that short article was mostly about HCoV-EMC, and I didn't see it reference MERS.
